components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/util/RemoteFetchConfigurationUtils.java

```diff
@@ -42,6 +43,9 @@
 
     private static final Log log = LogFactory.getLog(RemoteFetchConfigurationUtils.class);
 
+    public static final String REMOTE_FETCH_ENABLED = "RemoteFetch.FetchEnabled";
```

Comment:
Can we move these constants to org.wso2.carbon.identity.remotefetch.common.RemoteFetchConstants, Since Its public?

Reply:
Fixed in 75d6d85d8f1af2f68350c94373303f176e36dc36